### PR TITLE
Added default http headers

### PR DESCRIPTION
**Pull request by festinalente91**
- added default http header and replaced all previous hardcoded codes.

### CODE REVIEW

File: shared/actions/authActions.js [hunk 1]
```diff
@@ -310,14 +294,10 @@ export const requestRefreshTokenGuest = async (refreshToken) => {
...
-    const headerGetUserInformation = {
-      Accept: 'application/json',
-      'Content-Type': 'application/json',
-      Authorization: `bearer ${accessToken}`
-    };
+    const headers = getAuthHeaders(accessToken);
     return await fetch(address, {
       method: 'GET',
-      headers: headerGetUserInformation
+      headers
```
Comment (ochanje210) [inline shared/actions/authActions.js]: ok then, why don't you unify your code style?

Comment (festinalente91) [inline shared/actions/authActions.js]: np :)

**Comment by ochanje210** [comment]
awesome. thank you! lgtm

**Comment by yuda1124** [comment]
good work! LGTM

**Comment by ochanje210** [comment]
shipit

**Pull request merged**